src/sentry/integrations/github/integration.py

```diff
@@ -64,6 +64,9 @@
     ),
 ]
 
+# The order determines which location takes precidence when fetching CODEOWNERS files
+CODEOWNERS_LOCATIONS = ["CODEOWNERS", ".github/CODEOWNERS", "docs/CODEOWNERS"]
```

Comment:
This is good I prefer making lists like this a constant, but it looks like this variable is unread.

Reply:
Yeah I opted to keep them as fields on the integration since I moved the logic up to the mixin, but let me know if there's a better way to structure it all :+1: